cluster/gce/config-test.sh

```diff
@@ -95,8 +95,10 @@ fi
 NETWORK_PROVIDER="${NETWORK_PROVIDER:-cni}" # none, kubenet doesn't work with container runtime, update to cni and flannel
 
 if [[ "${NETWORK_PROVIDER:-}" == "mizar" ]]; then
-  NETWORK_PROVIDER_VERSION="0.9"
+  NETWORK_PROVIDER_VERSION="dev"
```

Comment:
This need to be changed when merging back to master

Reply:
@vinaykul Could you please explain this? I changed here to match config-test same as config-default.